### ISSUE TITLE
[featuresuggestion] Paging note sheet instead for scrolling

### ISSUE THREAD

**Issue by pedjas** [Bewelge/MIDIano#27]
Scrolling note sheet is quite hard to follow. It would be helpful if there is option to page instead.

That means notes would not scroll but cursor is moved and when it gets to the end of "page" it flips to new page and starts over.

**Comment by Bewelge** [comment]
![image](https://user-images.githubusercontent.com/17884995/221257212-9cdbf32d-9919-45ab-8923-bf01281b491e.png)
You can already disable the scrolling. But there's no page flipping animation which makes it admittedly not that great. Will mark as enhancement.

**Comment by pedjas** [comment]
I did look for it but  missed that option. Animation is not that necessary.

When scrolling is off, it works fine, exactly as expected. Only annoying is that cursor moves backwards to the beginning of page. It should just flip to the beginning.

**Comment by Bewelge** [comment]
Cursor now flips to the beginning
